.github/workflows/build-fast.yml

```
@@ -10,18 +10,30 @@ concurrency:
 
 jobs:
   linux:
-    name: ${{matrix.runner}}-${{matrix.compiler}}-${{matrix.version}}-llvm${{matrix.llvm}}
+    name: ${{matrix.runner}}-${{matrix.compiler}}-${{matrix.version}}-${{ matrix.shared == 'ON' && 'shared' || 'static' }}-llvm${{matrix.llvm}}
```

With a bool, this will simplify:
```suggestion
    name: ${{matrix.runner}}-${{matrix.compiler}}-${{matrix.version}}-${{ matrix.shared && 'shared' || 'static' }}-llvm${{matrix.llvm}}
```